ssg/build_remediations.py

```diff
@@ -231,31 +201,17 @@ def parse_from_file_with_jinja(self, env_yaml, cpe_platforms):
         if stripped_fix_text == "":
             return result
 
-        rule_specific_cpe_platform_names = set()
-        inherited_cpe_platform_names = set()
-        if self.associated_rule:
-            # There can be repeated inherited platforms and rule platforms
-            inherited_cpe_platform_names.update(self.associated_rule.inherited_cpe_platform_names)
-            if self.associated_rule.cpe_platform_names is not None:
-                rule_specific_cpe_platform_names = {
-                    p for p in self.associated_rule.cpe_platform_names
-                    if p not in inherited_cpe_platform_names}
-
-        inherited_conditionals = [
-            cpe_platforms[p].to_bash_conditional()
-            for p in inherited_cpe_platform_names]
-        rule_specific_conditionals = [
-            cpe_platforms[p].to_bash_conditional()
-            for p in rule_specific_cpe_platform_names]
-        # remove potential "None" from lists
-        inherited_conditionals = sorted([
-            p for p in inherited_conditionals if p != ''])
-        rule_specific_conditionals = sorted([
-            p for p in rule_specific_conditionals if p != ''])
-
+        inherited_conditionals = super(
+                                       BashRemediation,
+                                       self).get_inherited_conditionals(
+                                       "bash", cpe_platforms)
+        rule_specific_conditionals = super(
+                                           BashRemediation,
+                                           self).get_rule_specific_conditionals(
+                                           "bash", cpe_platforms)
         if inherited_conditionals or rule_specific_conditionals:
-            wrapped_fix_text = ["# Remediation is applicable only in certain platforms"]
-
+            wrapped_fix_text = ["# Remediation is applicable only in certain platforms",
+                                "do_something_magical"]
```

Comment:
????

Reply:
This got back somehow unfortunately during rebase. Fixed: https://github.com/ComplianceAsCode/content/pull/8923/commits/d5289e502c6b8c34a78ccac6585928646d9f840a